src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <section class="calendar-container">
+      <div class="calendar calendar--month-length--28 calendar--day--sun">
```

Comment:
Your BEM is good 👍 I guess `calendar--month-length_28` is the more correct syntax.